app/build.gradle

```diff
@@ -1,10 +1,12 @@
 apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
 
 android {
     compileSdkVersion 26
     defaultConfig {
         applicationId "com.mbcdev.goddamn"
-        minSdkVersion 24
+        minSdkVersion 23
```

Comment:
I think 24 is the minimum
https://developer.android.com/reference/android/service/quicksettings/Tile.html

Reply:
FINE